setup.cfg

```diff
@@ -2,5 +2,5 @@
 DJANGO_SETTINGS_MODULE=usaspending_api.settings
 
 [flake8]
-exclude=usaspending_api/*/migrations/*
+exclude=.git,.venv,usaspending_api/*/migrations/*
```

Comment:
We've never had issues without listing `.git`

Seems its excluded by default: http://flake8.pycqa.org/en/latest/manpage.html#options

Reply:
Fixed.